### ISSUE TITLE
android app: reports auth expired, but working just fine

### ISSUE THREAD

**Issue by fgergo** [tailscale/tailscale#469]
**Describe the bug**
installed from https://play.google.com/apps/testing/com.tailscale.ipn/join
after authentication
immediately error notification:
Your authentication has expired 
reauthenticate to maintain connection to ...
This notification can be deleted, but it comes back after ~1minute.

But app is working just fine:
on phone:
ping hello.ipn.dev # ok
ping 100.myhomewindowstailscaleip # ok
on myhomewindows:
ping 100.myandroidphonetailscaleip # ok

and another not deletable notification:
"Tailscale
Connected"

This is android 8.1.0
I'm not sure how to provide more detail.

**Expected behavior**
app should not complain if it's working fine.
(Of course this is much better than the the reverse would be :)

**Screenshots**
attached
![Screenshot_2020-06-17-23-45-15](https://user-images.githubusercontent.com/5156576/84954131-1ee4ab00-b0f5-11ea-8299-593644eae90d.png)

**Version information:**
 - lg android 8.1.0 (latest patch for this phone)

I can provide more info if it helps and I know what else to share.

**Comment by eliasnaur** [comment]
There is a small text description of your expiration date in the "3 dots" menu, just above "Copy My IP Address". Can you tell me what it says? If you can, please send me your username to mail@eliasnaur.com and I'll be able to access your tailscale logs for the device. Make sure you're running the latest beta, 0.1.10.

**Comment by fgergo** [comment]
When I switch the switch to "Active" the notification message appears immediately. The string is: "Expires: (never)". After I clear this notification it reappears after ~1 minute, and the string is the same: "Expires: (never)".

username sent.

Now I did a "Reathenticate", after the authentication the bogus notification appeared immediately. After I cleared it, it reappeared immediately, after that I cleared again. Thirdd bogus notification appeared in ~1minute again.

app version: 0.1.10-0-g676447a
